fe/src/main/java/org/apache/doris/load/Load.java

```diff
@@ -888,6 +888,10 @@ public static void initColumns(Table tbl, List<ImportColumnDesc> columnExprs,
             }
         }
 
+        // We make a copy of the columnExprs so that our subsequent changes
+        // to the columnExprs will not affect the original columnExprs.
+        List<ImportColumnDesc> copiedColumnExprs = Lists.newArrayList(columnExprs);
```

Comment:
seems that `copiedColumnExprs` is not used by anyone

Reply:
My bad, forgot to save the code...